pkgs/tools/misc/stow/default.nix

```diff
@@ -14,6 +14,7 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-p7rGc9qiRs9U/7SKET6t0gSmFWMOdrU7X2TYGidbd7M=";
   };
 
+  nativeBuildInputs = [ perlPackages.perl ];
```

Comment:
Wait so why do we need `perl` in the input arguments

Reply:
good catch! That's now removed.